plugin/storage/grpc/shared/grpc_handler.go

```diff
@@ -244,45 +241,16 @@ func (*GRPCHandler) sendSpans(spans []*model.Span, sendFn func(*storage_v1.Spans
 
 func (s *GRPCHandler) Capabilities(context.Context, *storage_v1.CapabilitiesRequest) (*storage_v1.CapabilitiesResponse, error) {
 	return &storage_v1.CapabilitiesResponse{
-		ArchiveSpanReader:   s.impl.ArchiveSpanReader() != nil,
-		ArchiveSpanWriter:   s.impl.ArchiveSpanWriter() != nil,
+		ArchiveSpanReader:   s.impl.SpanReader() != nil,
+		ArchiveSpanWriter:   s.impl.SpanWriter() != nil,
 		StreamingSpanWriter: s.impl.StreamingSpanWriter() != nil,
 	}, nil
 }
 
 func (s *GRPCHandler) GetArchiveTrace(r *storage_v1.GetTraceRequest, stream storage_v1.ArchiveSpanReaderPlugin_GetArchiveTraceServer) error {
```

Comment:
should this return not-implemented error instead?

Reply:
@yurishkuro are we okay with making that breaking change?

Reply:
this will never be called by grpc-storage

Reply:
@yurishkuro is there a reason we can't just remove this then?